### PR TITLE
#RI-3935 - Rework connections timeouts

### PR DESCRIPTION
**Pull request by zalenskiSofteq**

### CODE REVIEW

File: redisinsight/api/test/api/database/PUT-databases-id.test.ts [hunk 1]
```diff
@@ -156,18 +158,19 @@ describe(`PUT /databases/:id`, () => {
             port: constants.TEST_REDIS_PORT,
             username: null,
             password: null,
+            // timeout: null,
```
Comment (arthosofteq) [inline redisinsight/api/test/api/database/PUT-databases-id.test.ts:161]: why this commented?

File: redisinsight/api/test/api/database/PUT-databases-id.test.ts [hunk 2]
```diff
@@ -156,18 +158,19 @@ describe(`PUT /databases/:id`, () => {
...
             tlsServername: null,
           },
           after: async () => {
             newDatabase = await localDb.getInstanceById(constants.TEST_INSTANCE_ID_3);
-            expect(newDatabase).to.contain({
-              ..._.omit(oldDatabase, ['modules', 'provider', 'lastConnection', 'new', 'ssh']),
-              host: constants.TEST_REDIS_HOST,
-              port: constants.TEST_REDIS_PORT,
-            });
+            // expect(newDatabase).to.contain({
```
Comment (arthosofteq) [inline redisinsight/api/test/api/database/PUT-databases-id.test.ts:169]: and this

**Review by arthosofteq — APPROVED** [review]

File: redisinsight/api/test/api/database/PUT-databases-id.test.ts (hunk 2 above)
Comment (zalenskiSofteq) [reply redisinsight/api/test/api/database/PUT-databases-id.test.ts]: uncommented

File: redisinsight/api/test/api/database/PUT-databases-id.test.ts (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/api/test/api/database/PUT-databases-id.test.ts]: removed

**Pull request merged**